pages/app-layout/runtime-drawers.page.tsx

```diff
@@ -125,6 +125,19 @@ export default function WithDrawers() {
                 >
                   Resize circle3-global drawer to 500px
                 </Button>
+                <Button
+                  onClick={() => awsuiPlugins.appLayout.resizeDrawer('amazon-q', 700)}
```

Comment:
1. The drawer should have more generic name, not a trademark one
2. Does this button even work? Such id is not registered on this page